src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,252 @@
+| ANGACCChannelCount         | RECOMMENDED           | number        | Number of angular acceleration channels recorded by the system.                                                                                                                                     |
+| ANGVELChannelCount         | RECOMMENDED           | number        | Number of angular velocity channels recorded by the system.                                                                                                                                         |
+| JNTANGChannelCount         | RECOMMENDED           | number        | Number of joint angle channels recorded by the system.                                                                                                                                              |
+| MAGNChannelCount           | RECOMMENDED           | number        | Number of magnetic field strength channels recorded by the system.                                                                                                                                  |
+| ORNTChannelCount           | RECOMMENDED           | number        | Number of orientation channels recorded by the system.                                                                                                                                              |
+| POSChannelCount            | RECOMMENDED           | number        | Number of position channels recorded by the system.                                                                                                                                                 |
+| VELChannelCount            | RECOMMENDED           | number        | Number of linear velocity channels recorded by the system.                                                                                                                                          |
+| ExternalSoftwareVersions   | OPTIONAL              | string        | Names and   versions of additional software used for presentation or recording, other   than the software designated by the manufacture of the motion capture system   in field "SoftwareVersions”. |
+| Manufacturer               | OPTIONAL              | string        | Manufacturer of   the motion tracking system.                                                                                                                                                       |
+| ManufacturersModelName     | OPTIONAL              | string        | Manufacturer’s   designation of the motion tracking hardware model.                                                                                                                                 |
+
+Example:
+
+```JSON
+{
+	"InstitutionName": "Kiel University",
```

Comment:
it seems that json example formatting is overindenting making it inconsistent with other examples (use just two spaces) and hard(er) to fit into the page width

Reply:
Hopefully taken care of now :)